dnd5e-de.js

```diff
@@ -6,6 +6,8 @@ import NPC from '../../systems/dnd5e/module/actor/sheets/npc.js';
 // Module
 import Config from './src/config.js';
 import Converters from './src/converters.js';
+import Diaglog from './src/window_popup.js';
```

Comment:
```suggestion
import Dialog from './src/window_popup.js';
```
Oder falls das für eine Kollision sorgt, sowas wie `DND5EDEInfoDialog`.